### DISCUSSION TITLE
MetaMask faucet

### DISCUSSION THREAD

**Discussion by MuhammadAmmar24** [nehal2210/Blockchain-Engineer-Course#17] [Q&A]
How to add free ethers in the metamask wallet for developing and testing drops?

**Comment by Nelly1996** [comment]
Hello. I use this Faucet https://goerlifaucet.com/

**Reply by MuhammadAmmar24** [reply]
ERROR: "To receive Goerli ETH, your wallet must have a minimum balance of 0.001 ETH on mainnet. Please add ETH to your wallet and try again."

**Reply by alexeyl2014** [reply]
I buy test ETH. its easy way to get many gETH for test

**Comment by valensov** [comment]
I use the same faucet.

**Reply by fiiiilll1** [reply]
https://goerlifaucet.com/

**Reply by MuhammadAmmar24** [reply]
ERROR: "To receive Goerli ETH, your wallet must have a minimum balance of 0.001 ETH on mainnet. Please add ETH to your wallet and try again."

**Reply by valensov** [reply]
So, add some and try again. It's better have some ETH and use this faucet.

**Comment by olenkashull** [comment]
it will be easier to buy them through the testnet bridge than to mess with the faucets of free ethers

**Comment by BritvaBo** [comment]
I use the same faucet.

**Comment by nehal2210** [comment]
@MuhammadAmmar24 Goerli network is now deprecated, so its difficult to get goerli eth however you can use sepolia network and get eth from below faucet.
https://sepoliafaucet.com/
